docs/gethue/content/en/posts/2021-07-26-zero-click-hue-importer-drag-and-drop-a-file-auto-create-a-table.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Zero Click Hue Importer -- Drag and drop a file, auto create a table
```

Comment:
Please avoid -- and Hue in titles

https://medium.com/@mintmillerwrites/how-to-write-a-medium-title-that-rocks-397df72efa9a

Reply:
i.e. also fix https://gethue.com/blog/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table/ ?

e.g.

`Create a SQL table in your Data Warehouse from a local file in two clicks`

Reply:
Something like:
Create SQL tables on the fly with zero clicks